### PR TITLE
Adding the Markdown file for Git Test 04

### PR DESCRIPTION
**Pull request by thammer12499**
Pull Request for GitTest04. If someone could merge this I would greatly appreciate it, lol!

### CODE REVIEW

**Comment by kzp308** [comment]
I agree that the visuals are harsh. The red wording on the page just doesn't look right. I don't think the white looks as bad as the red though.

File: GitHub-Test-DHprojects/hammer_gitTest4.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+
+One of the problems I encountered immediately looking into the text analysis was the very subjective nature of the word labeling. This is generally just due to some words having different connotations when used in different contexts, I find this heavily applies to the "Gothic" label as many of the words within it can be easily applied into "other" or don't have much that relates to it aside from the general nature vaguely relating to the topic. It's more that these words can be found frequently in the Gothic genre rather than actually being of that nature.
+
+
+The style of the website leaves a bit to be desired and I believe can be capitalized on quite a bit. The heavy usage of dark background contrasting with the bright reds and white of the text can be quite straining on the eyes to read and generally makes it more difficult to follow. I would personally recommend a style more akin to an actual page of a book and focus on that, instead of trying to get across the dark atmosphere through dark green on top of blinding whites and reds.
```
Comment (julianjmg) [inline GitHub-Test-DHprojects/hammer_gitTest4.md:10]: Yeah I really did not like the layout. There werent many titles and the way the text was organized on the page was not quite right. And the font on the graph was hard to read.

**Review by julianjmg — APPROVED** [review]

**Review by kzp308 — APPROVED** [review]
I agree that the visuals are harsh. The red wording on the page just doesn't look right. I don't think the white looks as bad as the red though.

File: GitHub-Test-DHprojects/hammer_gitTest4.md (hunk 1 above)
Comment (kzp308) [inline GitHub-Test-DHprojects/hammer_gitTest4.md:10]: Not only this, but I thought the background sounds were also weird.

**Pull request merged**